src/Application.php

```diff
@@ -43,9 +43,7 @@ public static function main($binDir) {
     $application = new Application('cv', static::version() ?? 'UNKNOWN');
 
     $application->setAutoExit(FALSE);
-    ErrorHandler::pushHandler();
     $result = $application->run();
-    ErrorHandler::popHandler();
```

Comment:
I believe that this was a [response](https://github.com/civicrm/cv/pull/146) to shifts in how "Symfony Console" approached error handling. Cv used `symfony/console` 2.x for a long time, and this wasn't needed. (Maybe 2.x had a more "batteries included" philosophy? Or just didn't have the bug) Somewhere around 3.x or 4.x, the default error-handling became... worse ("less opinionated"? "more vanilla"?)... e.g. it would `exit(0)` even if there was an error.

Of course, now we've switched to 5.x... maybe it's changed again...

Reply:
Ok. I have had silence before this one existed, so it's probably not the biggest one, although there must be some error handler somewhere because normally `@` doesn't silence fatals. But removing the `@` is maybe the best bang for buck.